### PR TITLE
Update source code based on 3.0.0 release

### PR DESCRIPTION
**Pull request by tejas-haptik**
Version 3.0.0 of Haptik SDK brings breaking changes in the way we initialize the SDK

update-lib

### CODE REVIEW

File: app/src/main/java/ai/haptik/android/sample/app/AppApplication.java [hunk 1]
```diff
@@ -14,8 +14,6 @@ public void onCreate() {
             HaptikLib.setDebugEnabled(true);
```
Comment (findjigar) [inline app/src/main/java/ai/haptik/android/sample/app/AppApplication.java:14]: We don't need this here because it's being added in the `init` builder

**Review by findjigar — COMMENTED** [review]
keep in mind that you have not added offers screen implementation. Not sure if that's intentional to save time or not.

File: app/src/main/java/ai/haptik/android/sample/app/AppApplication.java (hunk 1 above)
Comment (tejas-haptik) [reply app/src/main/java/ai/haptik/android/sample/app/AppApplication.java:14]: Missed it. Will update it.

**Comment by tejas-haptik** [comment]
Well actually I just checked now, we dont have `UserSavedCards` implementation as well

**Review by findjigar — APPROVED** [review]

**Pull request merged**